dm/worker/worker.go

```diff
@@ -80,8 +80,14 @@ func NewWorker(cfg *Config) (*Worker, error) {
 	}
 	w.relayPurger = purger
 
-	// open kv db
+	// try upgrade from an older version
 	dbDir := path.Join(w.cfg.MetaDir, "kv")
+	err = tryUpgrade(dbDir)
+	if err != nil {
+		return nil, errors.Annotatef(err, "try to upgrade from any older version to %d", currentWorkerVersion)
```

Comment:
would output null or older version?

Reply:
`currentWorkerVersion` is the hardcoded version of the current binary. old version can't output this log, but new version always has a valid value.